pwndbg/heap/ptmalloc.py

```diff
@@ -15,7 +15,11 @@
 from pwndbg.constants import ptmalloc
 from pwndbg.heap import heap_chain_limit
 
-HEAP_MAX_SIZE     = 1024 * 1024
+HEAP_MAX_SIZE = 1024 * 1024 if pwndbg.arch.ptrsize == 4 else 2 * 4 * 1024 * 1024 * 8
```

Comment:
Can we provide any source links to that? e.g. a link to libc commit or some explanatory comment with glibc version etc

Reply:
Sure.